openedx/features/badging/models.py

```diff
@@ -43,12 +43,12 @@ class Badge(models.Model):
         badge_constants.TEAM_PLAYER
     )
 
-    name = models.CharField(max_length=255, blank=False, null=False)
-    description = models.TextField(blank=True, null=True)
-    threshold = models.PositiveIntegerField(blank=False, null=False)
-    type = models.CharField(max_length=100, blank=False, null=False, choices=BADGE_TYPES)
-    image = models.CharField(max_length=255, blank=False, null=False)
-    color_image = models.CharField(max_length=255, blank=False, null=False, default='')
```

Comment:
Refactored this model, as we don't need `blank=False, null=False` as both are by default False according to django documentation. 

https://docs.djangoproject.com/en/1.11/ref/models/fields/#null
https://docs.djangoproject.com/en/1.11/ref/models/fields/#blank